lambdas/src/handlers/access-token-handler.ts

```diff
@@ -36,7 +36,7 @@ export class AccessTokenLambda implements LambdaInterface {
             const requestPayload = this.requestValidator.validatePayload(event.body);
             const sessionItem = await this.sessionService.getSessionByAuthorizationCode(requestPayload.code);
             logger.appendKeys({ govuk_signin_journey_id: sessionItem.clientSessionId });
-            logger.info("found session")
+            logger.info("found session");
```

Comment:
This should be ideally debug or trace level logging. Consider making the message a bit more informative such as passing in the ID that was found

Reply:
I'm following our pattern of implementing as close as possible to the java lambdas, I agree in theory but this is from the java lambda https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/a2c1d73dca85643fe9d120c68d338f830476f5a8/accesstoken/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AccessTokenHandler.java#L59